src/hocon_pp.erl

```diff
@@ -184,7 +184,7 @@ is_quote_key(K) ->
 
 is_quote_str(S) ->
     case hocon_scanner:string(S) of
-        {ok, [{string, 1, S}], 1} ->
+        {ok, [{Tag, 1, S}], 1} when Tag =:= string orelse Tag =:= unqstr ->
```

Comment:
Q: the function name, at first glance, seems to indicate whether we have a quoted string, but here we are matching unquoted strings?  Perhaps the fn purpose is to decide whether to quote the input?